docs/src/get-started/goals.rst

```diff
@@ -0,0 +1,98 @@
+.. _userdoc-goals:
+
+Goals of this library
+=====================
+
+With the creation of equistore, we have three main use cases in mind:
+
+1. provide an interchange format for the atomistic machine learning ecosystem,
```

Comment:
Is `interchange format` a standardized term in the field? If not, it's a weird wording.

Reply:
It is kind of taking inspiration on https://en.wikipedia.org/wiki/Interchange_File_Format and friends. But I can find a better naming

Reply:
I got the reference, it just doesn't sound quite right grammatically, so unless it's a standard term, I would probably use "interchangeable" or something of the like